Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/02-state-storage.md

```diff
@@ -1,38 +1,37 @@
 # Use the Azure CLI to create a storage account to store the Terraform state files.
 This storage account will be used to store the state of each deployment step and will be accessed by Terraform to reference values stored in the various deployment state files.
 
-Create some variables to start with
-
-```bash
-REGION=<REGION>
-STORAGEACCOUNTNAME=<UNIQUENAME>
-CONTAINERNAME=akscs
-TFSTATE_RG=tfstate
-```
-
-or with Powershell
-```powershell
-$REGION=<REGION>
-$STORAGEACCOUNTNAME=<UNIQUENAME>
-$CONTAINERNAME=akscs
-$TFSTATE_RG=tfstate
-```
-
-Create a Resource Group:
-```
-az group create --name $TFSTATE_RG --location $REGION
-```
-
-Create a Storage Account:
-```
-az storage account create -n $STORAGEACCOUNTNAME -g $TFSTATE_RG -l $REGION --sku Standard_LRS
-```
-
-Create a Storage Container within the Storage Account:
-
-```
-az storage container-rm create --storage-account $STORAGEACCOUNTNAME --name $CONTAINERNAME
-```
+1. Login to the Azure subscription that you'll be deploying into with your credentials
+   ``` az login 
+       az account set --subscription <Your subscription ID> 
+   ```
+2. Create some variables to start with about where your storage account will live.
+    ```Powershell
+    $REGION=<REGION>
+    $STORAGEACCOUNTNAME=<UNIQUENAME>
+    $CONTAINERNAME=akscs
+    $TFSTATE_RG=tfstate
+    ```
+3. Create a Resource Group:
+    ```
+    az group create --name $TFSTATE_RG --location $REGION
+    ```
+
+4. Create a Storage Account:
+    ```
+    az storage account create -n $STORAGEACCOUNTNAME -g $TFSTATE_RG -l $REGION --sku Standard_LRS
+    ```
+
+5. Create a Storage Container within the Storage Account:
+
+    ```
+    az storage container-rm create --storage-account $STORAGEACCOUNTNAME --name $CONTAINERNAME
+    ```
+
+Key points:
+
+* For this example and for simplicity, public access is allowed to this Azure storage account for storing Terraform state. In a production deployment, it is recommended to retrict access to this storage account using a [storage firewall, service endpoint, or private endpoint](https://learn.microsoft.com/azure/storage/common/storage-network-security).
+* Azure storage accounts require a globally unique name. To learn more about troubleshooting storage account names, see [Resolve errors for storage account names.](https://learn.microsoft.com/azure/azure-resource-manager/templates/error-storage-account-name)
 
 ### Next step
```

Comment:
Below here is where the user first branches.  I think it would be good to pause and describe this choice here.  What happens when you go down one path vs the other. Are they exactly the same, other than shell choice?  (they aren't right now).  This is part of the last bullet of Step 1's feedback.  Make the learner feel confident to make this choice and comfortable that they aren't missing out on anything.

Also you'll need to address the Bash / PowerShell thing above, so many that'll influence what happens here.

Reply:
I updated all the instructions to only use PowerShell. For the next steps, I made it more clear the difference between import and create for the AAD groups. I also renamed the files so they make more sense when the user reaches that next step.

Reply:
That looks great, thanks!